### PR TITLE
docs: updated past changesets

### PR DESCRIPTION
**Pull request by bennypowers**
## What I did

1. Updated changsets to bring some consistency of voice and format

## Testing Instructions

1. no need to test, just read.

## Notes to Reviewers

Some of these changesets already went out, so they can be ignored. Some others we will likely remove immediately prior to releasing 1.0.

### CODE REVIEW

File: .changeset/fresh-comics-repeat.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 "@rhds/elements": minor
 ---
 
-Added `rh-tag`. 
+Added `<rh-tag>`.
 
 A tag is an inline-block element component that provides a distinct visual style for metadata in a UI.  Supports adding icon by attribute or slotted
```
Comment (heyMP) [inline .changeset/fresh-comics-repeat.md:7]:
```suggestion
A tag is an inline-block element component that provides a distinct visual style for metadata in a UI.  Supports adding icon by attribute or slotted.
```

Comment (heyMP) [reply .changeset/fresh-comics-repeat.md:7]: Not sure if we care about these or not.

File: .changeset/sharp-frogs-sing.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 "@rhds/elements": patch
 ---
 
-* Fixes missing font-family stacks when base css isn't applied
-* Adds rh-token CSS custom properties for font family stacks
-* Fixes regression in spacing for slotted cta
+css:
+  - added missing `font-family` stacks when base css isn't applied
+  - [Red Hat Design Tokens][tokens] for font family stacks
```
Comment (heyMP) [inline .changeset/sharp-frogs-sing.md:7]: Should this be a link?

Comment (heyMP) [reply .changeset/sharp-frogs-sing.md:7]: I see, it's a reference to the link below.  Is that because links look bad in raw Markdown?

**Review by heyMP — APPROVED** [review]
Looks good to me.  Consider my comments just questions.  Feel free to merge by resolving the comments.  📕 👏

File: .changeset/sharp-frogs-sing.md (hunk 2 above)
Comment (bennypowers) [reply .changeset/sharp-frogs-sing.md:7]: Six of one...

I like the shorter lines

**Pull request merged**